src/main/webapp/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="refresh" content="0; url=readfile.html" />
+    <meta http-equiv="refresh" content="0; url=login.html" />
   </head>
   <body>
   </body>
```

Comment:
IMO it makes more sense to have the login page on index.html rather then sending a redirect

Reply:
Either way, it could work. We just need an index.html

Reply:
Cool, I'll move login to index.html.